blueman/plugins/applet/TransferService.py

```diff
@@ -252,6 +281,7 @@ def _on_transfer_completed(self, _manager, transfer_path, success):
             now = datetime.now()
             filename = "%s_%s" % (now.strftime("%Y%m%d%H%M%S"), filename)
             dprint("Destination file exists, renaming to: %s" % filename)
+            dest = os.path.join(dest_dir, filename)
```

Comment:
It's not a duplicate.  `filename` changes in between this line and the prior one.  This is a bug that I am fixing, meant to actually enact the wishes of the three lines of code above it.

I will spin this off to a separate commit, so it won't be mixed with the feature commit.